pieman.sh

```diff
@@ -246,4 +337,8 @@ else
     image="${IMAGE}${extension}"
 fi
 
-success "${image} was built. Use Etcher (https://etcher.io) to burn it to your SD card."
+if ${CREATE_ONLY_ARTIFACT}; then
```

Comment:
Let's use present tense here. Similar to `Chroot environment ${R} is done.` for chroot,

Reply:
There is a corresponding issue https://github.com/tolstoyevsky/pieman/issues/122.